pages/flow/index.js

```diff
@@ -133,9 +134,8 @@ const StyledContainer = styled.section`
 
 const StyledCloseButton = styled.button`
   border: none;
-  background-color: #5d6bea;
-  color: #f5f5f5;
-  padding: 0;
+  background-color: var(--background-neutral);
+  color: var(--primary);
   width: 1.5rem;
   height: 1.5rem;
   border-radius: 50%;
```

Comment:
Are you sure you want to style the border radius in percentage?

Reply:
It's a circle. Can it cause any problems?

Reply:
Oh no sorry. Did not look at your vercel preview. Its of course perfect for a circle

Reply:
Thanks for the hint! I will check how it scales in the browser :)